### PR TITLE
osslsigncode 2.5

### PR DESCRIPTION
**Pull request by chenrui333**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/osslsigncode.rb [hunk 1]
```diff
@@ -14,22 +14,42 @@ class Osslsigncode < Formula
...
 
+  # Fix file INSTALL issue and install completion as part of `install` step
+  patch :DATA
+
   def install
-    system "./bootstrap"
-    system "./configure", *std_configure_args
-    system "make", "install"
+    # Fixing libcurl linker issue with `curl_version`
+    curl_lib = OS.mac? ? MacOS.sdk_path_if_needed/"usr/lib" : Formula["curl"].opt_lib
```
Comment (cho-m) [inline Formula/osslsigncode.rb:27]: `MacOS.sdk_path_if_needed` can be `nil`. Can either modify conditional or use string interpolation `#{MacOS.sdk_path_if_needed}/usr/lib"`

File: Formula/osslsigncode.rb [hunk 2]
```diff
@@ -14,22 +14,42 @@ class Osslsigncode < Formula
...
   uses_from_macos "curl"
 
+  # Fix file INSTALL issue and install completion as part of `install` step
+  patch :DATA
+
   def install
-    system "./bootstrap"
-    system "./configure", *std_configure_args
-    system "make", "install"
+    # Fixing libcurl linker issue with `curl_version`
```
Comment (carlocab) [inline Formula/osslsigncode.rb:26]: What is the linker issue? Has it been reported upstream?

File: Formula/osslsigncode.rb [hunk 3]
```diff
@@ -14,22 +14,42 @@ class Osslsigncode < Formula
...
-  depends_on "libtool" => :build
-  depends_on "pkg-config" => :build
-  depends_on "openssl@1.1"
+  depends_on "cmake" => :build
+  depends_on "openssl@3"
 
   uses_from_macos "curl"
 
+  # Fix file INSTALL issue and install completion as part of `install` step
+  patch :DATA
```
Comment (carlocab) [inline Formula/osslsigncode.rb:23]: Can we report this upstream too?

Comment (chenrui333) [reply Formula/osslsigncode.rb:23]: yeah, this was reported already

Comment (carlocab) [reply Formula/osslsigncode.rb:23]: Can you add a link to the upstream report in the comment?

File: Formula/osslsigncode.rb (hunk 2 above)
Comment (carlocab) [reply Formula/osslsigncode.rb:26]: Ping?

File: Formula/osslsigncode.rb (hunk 3 above)
Comment (carlocab) [reply Formula/osslsigncode.rb:23]: Ping?

File: Formula/osslsigncode.rb (hunk 2 above)
Comment (chenrui333) [reply Formula/osslsigncode.rb:26]: could not find the `curl_version`

File: Formula/osslsigncode.rb (hunk 3 above)
Comment (chenrui333) [reply Formula/osslsigncode.rb:23]: yeah, let me also file a separate report on the linker issue

File: Formula/osslsigncode.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/osslsigncode.rb:27]: this is actually adopted from the formula, what is your proposed fix?

File: Formula/osslsigncode.rb (hunk 2 above)
Comment (chenrui333) [reply Formula/osslsigncode.rb:26]: build failure

```
  "_curl_easy_strerror", referenced from:
      _add_timestamp in osslsigncode.c.o
  "_curl_slist_append", referenced from:
      _add_timestamp in osslsigncode.c.o
  "_curl_slist_free_all", referenced from:
      _add_timestamp in osslsigncode.c.o
  "_curl_version", referenced from:
      _main in osslsigncode.c.o
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

Comment (chenrui333) [reply Formula/osslsigncode.rb:26]: reported, https://github.com/mtrojnar/osslsigncode/issues/179

**Pull request closed without merging**